.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: julia
-os:
-    - osx
+matrix:
+    - os: osx
+      osx_image: xcode7
```

Comment:
7 might have been a bad choice if this is one they're going to retire https://blog.travis-ci.com/2016-09-15-new-default-osx-image-coming/